### ISSUE TITLE
Latest ANRC refuses to boot FGO

### ISSUE THREAD

**Issue by RayOfSpeed** [wDCat/ANRC#81]
Happened with 1.21.1, still happening with 1.22.0. On NOX 3.8.1.3, which was working previously until FGO updated from 1.20.0 and locked us out again (but giving error code 60 or 92 after booting and crashing).
Error screen: http://i.xomf.com/qzcfx.png
Unfortunately I can't read Chinese so I can't give you anything more than that. This was on a fresh install. Rather than crashing with errcode 60 or 92 after a boot, ANRC refuses to boot FGO in the first place. Haven't tried it with other apps. Not sure if it's a problem with my hardware settings for NOX or anything like that since, again, I can't read Chinese.

**Comment by darkzamurai** [comment]
What cm did you use?

**Comment by RayOfSpeed** [comment]
cm?

**Comment by darkzamurai** [comment]
Its because on the changelog the cm 13 was tested and it was working but the developer is still working until now.....

**Comment by RayOfSpeed** [comment]
What does 'cm' stand for, is what I'm asking.

**Comment by locke12456** [comment]
![image](https://cloud.githubusercontent.com/assets/1829977/25603745/f472c49a-2f30-11e7-8650-1de9540c8e06.png)

**Comment by darkzamurai** [comment]
Cyanogenmod version (cm 13 is a android marshmallow custom ROM)

**Comment by darkzamurai** [comment]
@locke12456 yeah right but I have a cm13 ROM and my tab is restarting after the game is open for a few minutes that's what I fix today....

**Comment by RayOfSpeed** [comment]
Oh, right. No, I'm not using cyanogenmod. Running it straight through xposed on NOX. What's bugging me is that it's refusing to boot FGO in the first place, instead of the root hide failing and FGO crashing with err 60/92.

**Comment by darkzamurai** [comment]
emulators are still in development @RayOfSpeed

**Comment by RayOfSpeed** [comment]
Obviously, but I'm interested that it's *just now* started pitching an error when it worked previously, new root hide methods aside. :v

**Comment by darkzamurai** [comment]
For me i tested CM13 based roms from Android 4.4 (Kitkat) to Android 6.0 (Marshmallow) but including Lollipop too it works but there's a problem the game is Super Duper Lag a delay of press one button like my screenshot on other (what called a chatroom here in this github) because i have a 1GB Ram (Damn) but i play this game since London Campaign it was long ago..... but the game is shity and shity each update..... nah i will sell my account but no one interest it.... i will migrate on my China and the upcoming US version (i wish that version is same as China's best performance ingame)

**Comment by manaphy0003** [comment]
that's kinda weird that anrc is just refusing to start F/GO might be the new anti emulator shit that even stops the app from playing at all

**Comment by RayOfSpeed** [comment]
@manaphy0003 Doesn't seem likely since the error message is an ANRC error message, not an FGO error message. If I boot FGO straight from the emulator, it gives the usual err92 for a rooted device.
